### DISCUSSION TITLE
Reflow of long checkbox and radio labels

### DISCUSSION THREAD

**Discussion by almilner** [indiana-university/rivet-source#427] [Feature and enhancement requests]
When checkbox and radio labels reflow they lack sufficient line height. 

So they look like this:
![image](https://user-images.githubusercontent.com/47157307/51934733-45a42380-23d2-11e9-9be9-86a3178140c2.png)
 
Would be nice if line height was increased, and maybe indentation added.

![image](https://user-images.githubusercontent.com/47157307/51934804-75ebc200-23d2-11e9-9e5d-2313af70cfbd.png)

Example showing checkboxes and radios:
https://codepen.io/almilner/pen/MLjZNg

**Comment by zttodd** [comment]
We plan to look at this in a future release. See #180 for other potential checkbox improvements.
